client/src/components/Event/stylesEvent.css

```diff
@@ -0,0 +1,17 @@
+.event-container {
```

Comment:
obvykle staci pojmenovat ten soubor napr. `styles.css` , neni treba pridavat to `Event` do nazvu